### DISCUSSION TITLE
How to set up GitLab merge requests with Hosted Weblate?

### DISCUSSION THREAD

**Discussion by peterkraume** [WeblateOrg/weblate#11249] [Q&A] [answered]
How do I  set up GitLab merge requests with [Hosted Weblate](https://hosted.weblate.org/)?

In the [documentation](https://docs.weblate.org/en/latest/vcs.html#gitlab-merge-requests) it says that I need to store `GITLAB_CREDENTIALS` in my settings. But obviously I have no access to `settings.py`.
I've searched the whole web interface but I found no way to enter my GitLab API token.

**Comment by nijel** [comment, marked as answer]
It has to be configured by site admins, AFAIK we've already handled your case.
